.github/workflows/releases.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             -f target_commitish=trunk \
             -q .body > CHANGELOG.md
         env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Hi, please revert the changes to our workflow files. They work correctly and we don't want to change something that already works.